tests/VCR/RelaxedRequestMatcherTest.php

```diff
@@ -45,4 +45,31 @@ public function testRelaxedRequestMatcherHost()
         $this->assertFalse(RequestMatcher::matchHeaders($actualRequest, $cleanRequest));
         $this->assertTrue(RelaxedRequestMatcher::matchHeaders($actualRequest, $cleanRequest));
     }
+
+    public function testRelaxedRequestMatcherBody()
+    {
+        $actualRequest = Request::fromArray(
+            array(
+                'method' => 'POST',
+                'url'    => 'http://example.com/api/v2',
+                'body'   => 'This is not secret, but this is SuperSecret',
+            )
+        );
+        $cleanRequest = Request::fromArray(
+            array(
+                'method' => 'POST',
+                'url'    => 'http://example.com/api/v2',
+                'body'   => 'This is not secret, but this is ',
+            )
+        );
+        RelaxedRequestMatcher::configureOptions(
+            array(
+                'bodyScrubber' => function ($body) {
```

Comment:
`RelaxedRequestMatcher::configureOptions()` takes `bodyScrubber` as an array, no? Shouldn't this be an array of functions instead?